src/openapi/apis/AuthApi.ts

```diff
@@ -0,0 +1,226 @@
+/* tslint:disable */
```

Comment:
Do we need this regenerated code in this PR?

Reply:
Yes we need the Playground models